src/effects/effects_base/effectstypes.h

```diff
@@ -62,6 +62,8 @@ constexpr const char16_t* EFFECT_OPEN_ACTION = u"action://effects/open?effectId=
 constexpr const char16_t* REALTIME_EFFECT_ADD_ACTION = u"action://effects/realtime-add?effectId=%1";
 constexpr const char16_t* REALTIME_EFFECT_REPLACE_ACTION = u"action://effects/realtime-replace?effectId=%1";
 
+constexpr const char16_t* EFFECT_VIEWER_URI = u"audacity://effects/effect_viewer?instanceId=%1&isVst=%2";
```

Comment:
It would be good to immediately keep in mind other types of effects, i.e. to use something like: 
```
// see enum class EffectFamily , use lower case 
constexpr const char16_t* EFFECT_VIEWER_URI = u"audacity://effects/effect_viewer?instanceId=%1&family=%2"
```

Reply:
I don't mind keep this for further development when some other family appears